code/src/Components/MessageList.css

```diff
@@ -0,0 +1,49 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+.message-wrapper{
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.message{
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  margin-bottom: 5vh;
+  border-style: solid;
+  box-shadow: 5px 7px;
+}
+
+h2{
+  font-family: 'Inconsolata', monospace;
+  margin: 15px;
+  word-wrap: break-word;
+}
+
+.heart-time{
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.heart{
+  margin-right: 35%;
+}
+
+@media screen and (min-width:768px) {
+  .message {
+      width: 70%;
+  }
+}
+
+@media screen and (min-width:1024px) {
```

Comment:
Another way to tackle this would be to specify a max-width, on the .message css rule